dashboards/views.py

```diff
@@ -216,3 +218,58 @@ def api_overview(request):
     }
 
     return render(request, "dashboards/api.html", {"context": context})
+
+
+def people_list(request):
+    state_people_data = {}
+
+    for state in states:
+        state_people_data[state.abbr.lower()] = {
+            "state": state.name,
+        }
+
+    return render(
+        request,
+        "dashboards/people_listing.html",
+        {"state_people_data": state_people_data},
+    )
+
+
+def people_matcher(request, state):
+    jid = abbr_to_jid(state)
+    all_sessions = sessions_with_bills(jid)
+    if all_sessions:
+        session = all_sessions[0]
+        unmatched = UnmatchedName.objects.filter(session_id=session)
+        state_sponsors = Person.objects.filter(current_jurisdiction_id=jid)
+
+    context = {
+        "state": state,
+        "session": session,
```

Comment:
getting an UnboundLocal here if session isn't set by the all_sessions conditional.  It's a weird edge case since there are no states without sessions in production of course, but maybe worth a small change.